enhancements/operator-conditions.md

```diff
@@ -118,6 +123,14 @@ The `Upgradeable` condition might be useful when:
 - An operator is about to start a critical process and should not be upgraded until after the process is completed.
 - The operator is performing a migration of CRs that must be completed before the operator is ready to be upgraded.
 
+#### Ready
+
+The `Ready` "OLM Supported Condition" allows an operator to communicate its readiness to OLM. By default, OLM will move a CSV in the `Succeeded` Phase to the `Failed` phase if the deployments [Readiness Probe](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes) is not true.
+
+The `Ready` condition might be useful when:
+
+- An operator wants to change its Readiness Probe status to 'false' to communicate information to a cluster admin but does not want OLM to move the operator into the failed state.
```

Comment:
What OLM behavior changes if `Ready=true` vs. `Ready=false`?

I understood "supported conditions" to be things that would affect OLM behavior (i.e. Upgradeable=false prevents upgrade)

That's not to say an author couldn't write a `Ready` status. But it's more likely useful for the operator to communicate more detailed status via its own CRs (i.e. connection state to cloud accounts).

Reply:
Evan and I discussed this condition offline and ultimately reached the conclusion that this condition should be removed.